src/styles/main.scss

```diff
@@ -4,8 +4,10 @@ body {
   margin: 0;
 
   display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 200px;
```

Comment:
For instance, if we have the following HTML:
```html
<div>
  hello world
</div>
```
Then we can do the replacement by writing:
```js
const div = document.querySelector("div");
const span = document.createElement("span");
span.innerHTML = "hello james";
div.parentNode.replaceChild(span, div);
```